src/game/clients/CClientEvent.cpp

```diff
@@ -422,9 +421,6 @@ void CClient::Event_Item_Drop( CUID uidItem, CPointMap pt, CUID uidOn, uchar gri
 			}
 		}
 
-		if ( pOldCont != pItem->GetContainer() )
```

Comment:
I remember adding this for a reason, does git blame help for that?

Reply:
i blame it and see that doesnt change in years
https://github.com/Sphereserver/Source-X/blame/ae696679c35edce7192b8fbb55350ba28003718d/src/graysvr/CClientEvent.cpp

Reply:
This check should be there, because this check if the item changed the container under dropon_item trigger, I mean think like 2 players tries to get same item, and you did some checks under dropon_item, for the first player, then added the item in his container, you don't want to second player take it away from player's backpack. This check is added to block this, but it probably could be really really rare issue, as sphere can handle these checks in milliseconds, but as someone can have tons of checks under @DropOn_Item, checking if the old container is same should be guarantee. So it has to stay like it was honestly.